packages/core/ios/RNEmbraceCore/SpanRepository.swift

```diff
@@ -6,9 +6,9 @@ import OpenTelemetryApi
 private let MAX_STORED_SPANS = 10000
 
 class SpanRepository {
-    private let activeSpansQueue = DispatchQueue(label: "io.embrace.RNEmbrace.activeSpans",
```

Comment:
btw I think all this will be transparent to the user but let's remember to do a test during QA for next release where we try upgrading an app from a previous version to one with the new name and see if anything weird happens